Detectors/ITSMFT/MFT/workflow/include/MFTWorkflow/ClustererSpec.h

```diff
@@ -37,9 +37,13 @@ class ClustererDPL : public Task
   void run(ProcessingContext& pc) final;
 
  private:
+  void updateTimeDependentParams(ProcessingContext& pc);
+  void finaliseCCDB(ConcreteDataMatcher& matcher, void* obj) final;
```

Comment:
It should not be private either.

Reply:
I confirm that if it's private it will not work, because the public, default, version has precedence in the lookup I do to see if the function is implemented in the baseclass.

Reply:
Thanks, yes, with public it works. I still have problem to get it for all devices where I expect this, but it seems to be related to whole FST workflow getting stuck (due to the PR8275). Now rebuilding with PR8275 reverted.

Reply:
Cool. I will try to see if there is a way to at least notice if there is a private method and complain.